src/util/options.schema.json

```diff
@@ -989,8 +989,21 @@
               "description": "Number of unique thread IDs allocated for each pthread_create node.",
               "type": "integer",
               "default": 0
+            },
+            "context": {
+              "title": "ana.thread.context",
+              "type": "object",
+              "properties": {
+                "create-edges": {
+                  "title": "ana.thread.context.create-edges",
+                  "description": "threadID analysis: Encountered create edges in context.",
```

Comment:
It's not just the context though but also the local state. So why not have it simply as `ana.thread.create-edges`?

Reply:
The set needs to be tracked anyway, the question really is whether they get put into the context or not.